core/nodejsActionBase/src/service.js

```diff
@@ -15,16 +15,21 @@
  * limitations under the License.
  */
 
+
 var NodeActionRunner = require('../runner');
 
-function NodeActionService(config) {
+function NodeActionService(cfg) {
+
     var Status = {
         ready: 'ready',
         starting: 'starting',
         running: 'running',
         stopped: 'stopped'
     };
 
+    var config = cfg;
```

Comment:
why bother with this? the value is available already to the entire class.

Reply:
updated to use `config` which is passed as a function parameter